llama-index-integrations/vector_stores/llama-index-vector-stores-qdrant/llama_index/vector_stores/qdrant/base.py

```diff
@@ -661,8 +662,8 @@ async def _acreate_collection(self, collection_name: str, vector_size: int) -> N
             index=rest.SparseIndexParams(),
             modifier=(
                 rest.Modifier.IDF
-                if self.fastembed_sparse_model and "bm42" in self.fastembed_sparse_model
-                else rest.Modifier.NONE
+                if self.fastembed_sparse_model in IDF_EMBEDDING_MODELS
+                else None
```

Comment:
Just a side note, but why is rest.Modified.NONE there if it's not handled by qdrant? 👀

Reply:
`rest.Modifier.None` has a conversion bug when using gRPC.

`qdrant-client` 1.11.1 will fix it.

Reply:
https://github.com/qdrant/qdrant-client/pull/748